internal/schemas.ts

```diff
@@ -1161,6 +1200,12 @@ const ImageAnnotations: FoxgloveMessageSchema = {
       description: "Points annotations",
       array: true,
     },
+    {
+      name: "labels",
```

Comment:
```suggestion
      name: "texts",
```
Matches SceneEntity.